carWarGame.py

```diff
@@ -37,6 +37,7 @@ class Deck:
 
     def __init__(self) -> None:
         self.cards = self.generate_deck()
+        self.length = len(self.cards)
```

Comment:
@andresgamo this is not dynamically updated, so IMO this is wrong.

Reply:
you can create a magic method for your Deck obj to get len of deck  `len(Deck())`

```
class Deck....
.
.
.
    def __len__(self):
        return len(self.cards)
.
.
.
```

Reply:
Yes! I realized that after I pushed it. This is what we were talking about last time. I am going to take a look at the magic method.